src/EntityFramework/EntityFramework.InternalsVisibleTo.csproj

```diff
@@ -5,6 +5,9 @@
     <RootNamespace>System.Data.Entity</RootNamespace>
     <TargetFrameworks>net45;netstandard2.1</TargetFrameworks>
     <AssemblyVersion>6.0.0.0</AssemblyVersion>
+
+    <!-- Properties\Resources.tt expects a path that does not match the class in Properties\Resources.cs -->
+    <EmbeddedResourceUseDependentUponConvention>false</EmbeddedResourceUseDependentUponConvention>
```

Comment:
I'd rather not disable this everywhere. Can we work around the issues it causes in a different way?

Reply:
Never mind, I like this--it makes netstandard2.1 consistent with net40 and net45